modules/objdetect/test/test_qrcode.cpp

```diff
@@ -481,6 +481,23 @@ INSTANTIATE_TEST_CASE_P(/**/, Objdetect_QRCode_Close, testing::ValuesIn(qrcode_i
 INSTANTIATE_TEST_CASE_P(/**/, Objdetect_QRCode_Monitor, testing::ValuesIn(qrcode_images_monitor));
 INSTANTIATE_TEST_CASE_P(/**/, Objdetect_QRCode_Multi, testing::ValuesIn(qrcode_images_multiple));
 
+TEST(Objdetect_QRCode_decodeMulti, decode_regression_16491)
+{
+#ifdef HAVE_QUIRC
+    Mat zero_image = Mat::zeros(256, 256, CV_8UC1);
+    std::vector<Point> vec_corners{Point(16, 16), Point(128, 16), Point(128, 128), Point(16, 128),
+                                   Point(16, 16), Point(128, 16), Point(128, 128), Point(16, 128)};
```

Comment:
C++11 is not allowed in OpenCV tests on 3.4 branch by default.
Please use `Points corners_[] = {...};` array + `vector::assign()`